dist/tools/doccheck/exclude_simple

```diff
@@ -8424,3 +8424,39 @@ warning: Member LSM6DSXX_PARAM_GYRO_FIFO_DEC (macro definition) of file lsm6dsxx
+warning: Member LSM6DSL_REG_SENSORHUB6_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENSORHUB7_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENSORHUB8_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENSORHUB9_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENSORHUB10_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENSORHUB11_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENSORHUB12_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENSORHUB13_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENSORHUB14_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENSORHUB15_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENSORHUB16_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENSORHUB17_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENSORHUB18_REG (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_FUNC_SRC_2 (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_WRIST_TILT_IA (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_MASTER_CMD_CODE (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_SENS_SYNC_SPI_ERR_CODE (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_OUT_MAG_RAW_X_L (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_OUT_MAG_RAW_X_H (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_OUT_MAG_RAW_Y_L (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_OUT_MAG_RAW_Y_H (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_OUT_MAG_RAW_Z_L (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_OUT_MAG_RAW_Z_H (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_X_OFS_USR (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_Y_OFS_USR (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DSL_REG_Z_OFS_USR (macro definition) of file lsm6dsxx_internal.h is not documented.
+warning: Member LSM6DS33_REG_ORIENT_CFG_G (macro definition) of file lsm6dsxx_internal.h is not documented.
```

Comment:
Is there no way of telling doxygen that a define is only used internally? Something like `@internal`?

Reply:
probably but I don't know how... this is what we did for the others.

Reply:
Would it make sense to just skip internal header files for the generated documentation? ala https://stackoverflow.com/questions/11025971/separate-internal-from-external-documentation-in-doxygen with `@cond`

or just skip those for the "missing documentation" check?